api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java

```diff
@@ -17,6 +17,7 @@
 package io.opentelemetry.metrics;
 
 import io.opentelemetry.metrics.DoubleCounter.BoundDoubleCounter;
+import io.opentelemetry.metrics.InstrumentWithBinding.BoundInstrument;
```

Comment:
intentional?

Reply:
It is. Oddly, IDEA wouldn't properly compile without it, even though it would compile with gradle. I'm honestly not sure what's going on, but this seems to be required by some versions of the compiler.

Reply:
Oh, that's strange. I checked out your branch and removed the lines to check if those are necessary, my IDEA compiled it without any complaints.

Reply:
What version of javac is your IDEA set up to use? I notice that IDEA is using 1.8.0_222 to compile, but when I use gradle, I'm using 1.8.0_232.

Reply:
For otel-java I've set it up to use `1.8.0_192`. Maybe you want to give it a try and see if the CI takes it without the unused imports?

Reply:
CI was working previous to this PR, so I'm guessing it does work. Is your IDEA using gradle for the compiles, or the internal compiler?

Reply:
I suppose it's using the JDK mentioned above which I set as _Project SDK_ in the project settings. For running Gradle itself (the daemon) it uses the JRE defined in the global IDEA settings (Build > Build Tools > Gradle > Gradle JVM).